soc/nordic/nrf53/nrf53_cpunet_mgmt.c

```diff
@@ -66,3 +68,33 @@ void nrf53_cpunet_enable(bool on)
 
 	__ASSERT_NO_MSG(ret >= 0);
 }
+
+#ifdef CONFIG_SOC_NRF53_CPUNET_ENABLE
+static int nrf53_cpunet_init(void)
+{
+	/* Route Bluetooth Controller Debug Pins */
+	DEBUG_SETUP();
```

Comment:
do you need that here? `CONFIG_SOC_NRF53_CPUNET_ENABLE` depends on `!BT` so it seems strange that bluetooth debug pins are set up here.

Reply:
hmm you're right, good catch